src/components/Annotation/Annotation.js

```diff
@@ -71,44 +78,51 @@ class Annotation extends React.PureComponent<Props> {
         };
 
         return (
-            <div
-                className={classNames(`ba-annotation ${className}`, {
-                    'ba-is-pending': isPending || error
-                })}
-                onBlur={onBlur}
-                onFocus={onFocus}
-            >
-                <div className='ba-annotation-content'>
-                    <Avatar className='ba-annotation-avatar' {...createdBy} />
-                    <div className='ba-annotation-text'>
-                        <div className='ba-annotation-headline'>
-                            <UserLink className='ba-annotation-user-name' {...annotator} />
-                            <Tooltip
-                                text={
-                                    <FormattedMessage
-                                        {...messages.annotationPostedFullDateTime}
-                                        values={{ time: createdAtTimestamp }}
+            <Internationalize language={language} messages={intlMessages}>
+                <div
+                    className={classNames(`ba-annotation ${className}`, {
+                        'ba-is-pending': isPending || error
+                    })}
+                    onBlur={onBlur}
+                    onFocus={onFocus}
+                >
+                    <div className='ba-annotation-content'>
+                        <Avatar className='ba-annotation-avatar' {...createdBy} />
+                        <div className='ba-annotation-text'>
+                            <div className='ba-annotation-headline'>
+                                <UserLink className='ba-annotation-user-name' {...annotator} />
+                                {createdAtTimestamp && (
```

Comment:
Seems like some further componentization could be helpful here.

Reply:
I can clean this up in a separate PR

Reply:
https://github.com/box/box-annotations/pull/218